core/src/main/scala/com.snowplowanalytics.stream/loader/transformers/JsonTransformer.scala

```diff
@@ -41,68 +38,53 @@ import com.snowplowanalytics.snowplow.analytics.scalasdk.json.EventTransformer._
  * Class to convert successfully enriched events to EmitterInputs
  *
  * @param documentIndex the elasticsearch index name
- * @param documentType the elasticsearch index type
  */
-class SnowplowElasticsearchTransformer(documentIndex: String, documentType: String)
-    extends ITransformer[ValidatedRecord, EmitterInput]
+class JsonTransformer(documentIndex: String)
```

Comment:
I think the rename is confusing wrt the `PlainJsonTransformer`, maybe `EnrichedEventJsonTransformer` wdyt?

Reply:
but all of them are `EnrichedEventJsonTransformer`

Reply:
renamed, although I am not fan of long names. you are the boss